src/app/device-creation/device-creation.component.html

```diff
@@ -27,9 +27,12 @@ <h2 class="text-center">Add device</h2>
                 [(ngModel)]="device.references"
                 placeholder="Publications describing (the methods behind) the device"
               ></textarea>
-              <label for="image">Image:</label>
-              <input type="file" class="form-control input-form border-0" id="image" name="image" (change)="handleFileInput($event.target.files)" />
-
+              <label for="image">Image: (MAX {{ fileSizeLimit }}KB) </label>
+              <input type="file" class="form-control input-form border-0" id="image" name="image"
+              [(ngModel)]="device.image" (change)="handleFileInput($event.target.files)" accept="{{ fileTypesAllowed }}"/>
+              <div class="error-container" [hidden]="!this.imageError">
```

Comment:
I think that it would be better to use a `ngIf`, or at least, that would be more consistent to, for example, https://github.com/icebreaker-science/frontend/blob/07f6b283251b124b49bc19a2bd7007559f70c54a/src/app/login/login.component.html#L6.